### PR TITLE
Update README for CentOS 7 - Duplication and SCL

### PR DESCRIPTION
**Pull request by caruccio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -229,11 +187,58 @@ make
 sudo make install
 ```
 
-Copy config file, the service and enable it:
+### CentOS 7
 
+_Note: As of April 2019, the service does not start on CentOS 7, see [#182](https://github.com/MisterTea/EternalTerminal/issues/182)_
```
Comment (MisterTea) [inline README.md]: Can we delete this comment now?

Comment (caruccio) [reply README.md]: done!

**Comment by MisterTea** [comment]
Thanks!

**Comment by MisterTea** [comment]
Thanks!

**Pull request merged**